README.md

```diff
@@ -44,8 +44,8 @@ For a guided tour, take a look at the [quick start
 guide](https://grpc.io/docs/languages/java/quickstart) or the more explanatory [gRPC
 basics](https://grpc.io/docs/languages/java/basics).
 
-The [examples](https://github.com/grpc/grpc-java/tree/v1.52.1/examples) and the
```

Comment:
Oh, so the 1.53 release process hadn't been completed. Oops. Nothing to do about it now.